.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "chrome",
+      "request": "attach",
+      "name": "Attach to Karma test:debug",
+      "address": "localhost",
+      "port": 9765, // keep in sync with debugPort in karma.conf.js
```

Comment:
Does .vscode not mind the comment in a json file?

Reply:
> Does .vscode not mind the comment in a json file?

Reply:
Yes, it supports this. https://code.visualstudio.com/docs/languages/json#_json-with-comments